contracts/Makefile

```
@@ -51,6 +51,16 @@ test:
 bench:
 	cargo test benchmark_
 
+# Run benchmarks and emit JSON output for CI diffing
+bench-json:
```

checkmake (0.3.2)

[warning] Target body for "bench-json" exceeds allowed length of 5 lines (7).

(maxbodylength)